config/testdata/example_config.yaml

```diff
@@ -58,4 +63,5 @@ map_rule:
         - "system:serviceaccount:" 
         - "system-serviceaccount-"
       athenz_user_prefix: user.
+      athenz_service_account_prefix: _kaas_namespace_.k8s._k8s_cluster_2._namespace_.service_account.
```

Comment:
Previously, service_domain was used as a prefix for serviceaccount.
I Added prefix for serviceaccount due to support multiple domains.